src/pybind/mgr/dashboard/tests/test_rbd_mirroring.py

```diff
@@ -71,7 +71,7 @@ def setup_server(cls):
 
     @mock.patch('dashboard.controllers.rbd_mirroring.rbd')
     def test_default(self, rbd_mock):  # pylint: disable=W0613
-        self._get('/api/test/rbdmirror')
+        self._get('/api/test/rbdmirror/__call__')
```

Comment:
oops, I forgot to update these tests. The idea is to change the cherrypy setup of controllers in the beginning of the test to avoid adding `__call__` to the URL.